src/pathways/common/services/lms-client.service.ts

```
@@ -535,4 +535,75 @@ export class LmsClientService {
+      const result = res.data?.result;
+      if (!result) {
+        return null;
+      }
+
+      const pricing = result.pricing;
+      const resolvedCourseId = result.courseId || courseId;
+      if (!pricing || pricing.amount === undefined || pricing.amount === null) {
+        return {
+          courseId: resolvedCourseId,
+          amount: 0,
+          currency: String(pricing?.currency || 'USD')
```

ESLint

[error] Replace `'USD'` with `"USD"`

(prettier/prettier)